src/cards/repo-card.js

```diff
@@ -61,6 +61,7 @@ const renderRepoCard = (repo, options = {}) => {
     isTemplate,
     starCount,
     forkCount,
+    openGraphImageUrl,
```

Comment:
I think you should add this as an optional option to the pin card:

https://github.com/anuraghazra/github-readme-stats/blob/ff169849880e3c2b0d89cc8d95b6542e71fc1906/api/pin.js#L13-L27